app/client/packages/storybook/src/components/ColorTable.tsx

```
@@ -23,7 +23,7 @@ export const ColorTable = ({
   const [isDarkMode, setDarkMode] = useState<boolean | undefined>(false);
   const { theme } = useTheme({
     seedColor,
-    colorMode: isDarkMode != null ? "dark" : "light",
+    colorMode: Boolean(isDarkMode) ? "dark" : "light",
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)